src/WebJobs.Script/HttpMethodJsonConverter.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Net.Http;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public class HttpMethodJsonConverter : JsonConverter
+    {
+        public override bool CanConvert(Type objectType)
+        {
+            if (objectType == typeof(HttpMethod))
+            {
+                return true;
+            }
+            return false;
+        }
+
+        public override object ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+        {
+            if (reader == null)
+            {
+                throw new ArgumentNullException("reader");
```

Comment:
It addresses an fxcop error. We can discuss outside of this review - if we want to remove that fxcop check then we won't get such errors anymore.